src/components/globalConfigurations/GlobalConfiguration.tsx

```diff
@@ -295,91 +317,127 @@ function NavItem({ hostURLConfig, serverMode }) {
+                                            className={`cursor ${
+                                                collapsedState[route.name] ? '' : 'fw-6'
+                                            } flex dc__content-space`}
+                                            onClick={(e) => {
+                                                handleGroupCollapsedState(e, route)
+                                            }}
+                                        >
+                                            {route.name}
+                                            <Dropdown
+                                                className="icon-dim-24 rotate"
+                                                style={{
+                                                    ['--rotateBy' as any]: !collapsedState[route.name]
+                                                        ? '180deg'
+                                                        : '0deg',
+                                                }}
+                                            />
+                                        </NavLink>
+                                        {!collapsedState[route.name] && (
+                                            <>
+                                                {route.group.map((_route) => {
+                                                    return renderNavItem(_route, 'ml-10', true)
+                                                })}
+                                            </>
+                                        )}
+                                    </>
+                                ) : (
+                                    renderNavItem(route)
+                                )),
+                        )}
+                        <hr className="mt-8 mb-8 w-100 checklist__divider" />
+                        <NavLink
+                            to={URLS.GLOBAL_CONFIG_EXTERNAL_LINKS}
+                            key={URLS.GLOBAL_CONFIG_EXTERNAL_LINKS}
+                            activeClassName="active-route"
+                        >
+                            <div className="flexbox flex-justify">External Links</div>
+                        </NavLink>
+                    </>
+                </>
             )}
-            <hr className="mt-8 mb-8 w-100 checklist__divider" />
-            <NavLink
-                to={URLS.GLOBAL_CONFIG_EXTERNAL_LINKS}
-                key={URLS.GLOBAL_CONFIG_EXTERNAL_LINKS}
-                activeClassName="active-route"
-            >
-                <div className="flexbox flex-justify">External Links</div>
-            </NavLink>
         </div>
     )
 }
 
 function Body({ getHostURLConfig, checkList, serverMode, handleChecklistUpdate }) {
     const location = useLocation()
 
+    const defaultRoute = (): string => {
+        if (window._env_.K8S_CLIENT) {
+            return URLS.GLOBAL_CONFIG_CLUSTER
+        } else if (serverMode && serverMode === SERVER_MODE.EA_ONLY) {
```

Comment:
I think keeping `serverMode === SERVER_MODE.EA_ONLY` will do it.

Reply:
done